### ISSUE TITLE
Proper license

### ISSUE THREAD

**Issue by adelevie** [adelevie/walverine#5]

**Comment by fbennett** [comment]
We seem to be on track for BSD. Mike plans to make the adjustment on CourtListener, and the license can be mapped through as appropriate on Walverine. I'll be happy to make a pull request covering the change.

**Comment by adelevie** [comment]
Great. Thanks, @fbennett.

**Comment by mlissner** [comment]
Looks like you've got the Affero license as of now. That's the strictest that we've got on any of our code. If you leave that, you'll be all set and rock solid, legally speaking.

I just re-licensed constants.py, so it is BSD as of a moment ago. I don't know how much code other than that you guys have from CL, but if that's all you have, you could license this as BSD. If not, then Affero is probably more appropriate. 

I'm considering relicensing the rest of the citation code as BSD, but I don't think I can do so with out the developer's consent. Besides that though, I like the Affero license, as it gives us protection against closed-source usages of the code (a good thing!)

**Comment by mlissner** [comment]
Oh, and thanks for taking this seriously. We do appreciate it.

**Comment by adelevie** [comment]
Not a problem. I'm just a caretaker of your great work, and I'm happy to accommodate your licensing needs.

**Comment by mlissner** [comment]
Excellent. Glad license-talk can be so easy, and thanks for the awesome tool. 

I'm tempted to add it to our API, actually, but haven't had a chance to check it out properly. A few people have already asked for something like this in API form.

**Comment by adelevie** [comment]
What would be added to the API? User POSTs text, API returns citations?

**Comment by mlissner** [comment]
Something like that, yeah.

**Comment by adelevie** [comment]
:thumbsup: 

Though it still is always nice to be able to run such a potentially-processor-intensive function on my own equipment.

**Comment by mlissner** [comment]
Exactly...hence our not having done it just yet despite people wanting
it....

**Comment by konklone** [comment]
Just wanted to see if there was any actionable stuff here. Is preventing closed-source use of the citator really a good thing? I'd love to get closed-source companies to use this citator, and mine, and for that to lead to their contribution and investment in the code -- thus improving it for all the open-source uses.
